src/store/flow/block-types/SmartDevices_LocationResponseBlockStore.ts

```diff
@@ -34,12 +35,22 @@ const actions: ActionTree<IEmptyState, IRootState> = {
       prompt: blankMessageResource.uuid,
       accuracy_threshold_meters: 5.0,
       accuracy_timeout_seconds: 120,
+      ...await dispatch('initiateExtraVendorConfig'),
     }
+
+    props.exits = [
+      await dispatch('flow/block_createBlockDefaultExitWith', {
+        props: ({
+          uuid: await (new IdGeneratorUuidV4()).generate(),
```

Comment:
Not a review comment, but rather a general thought. Would it be better to reuse an instance of generator from `flow-runner` context?

By importing `IdGeneratorUuidV4` explicitly, we can possibly override user's preference on ID generation:

https://github.com/FLOIP/flow-runner/blob/master/src/flow-spec/IContext.ts#L137

Besides, as the `generate()` is not static, the generator is semantically allowed to have some state, so creating new instances of it may lead to duplicates:

```ts
class IdGeneratorSimple implements IIdGenerator {
  let id = 0
  async generate(): Promise<string> {
    return ++this.id
  }
}
```

Reply:
Thanks @object-required for you input here, sorry I don't really understand the proposal :D .
Currently, we're using `IdGeneratorUuidV4` https://github.com/FLOIP/flow-runner/blob/e7a3ecf5d27bc941bff70dc69a5ffa578fc227cf/src/domain/IdGeneratorUuidV4.ts#L28 
```
export class IdGeneratorUuidV4 implements IIdGenerator {
  async generate(): Promise<string> {
    return uuid.v4()
  }
}
```
which provides a very unique uuid for every generation with `uuid.v4()`, so no duplicates expected !

(moving this PR to QA, but let's have a call about this if needed ;) )